### PR TITLE
updated Unity SDK Connect Wallet

### PR DESCRIPTION
**Pull request by ddocs**
- Amended Connect Wallet & Authenticate section

### CODE REVIEW

File: docs/Game/Unity/getting-started/01-connect.md [hunk 1]
```diff
@@ -3,25 +3,56 @@ title: 01 - Connect Wallet and Authenticate
 id: game-01
 ---
 
+Connecting to a Web3 wallet such as MetaMask via **WalletConnect** provides a link between a **Wallet Address** and a user's **Game Account**. 
 
-Connecting to a Web3 i.e. MetaMask wallet via **WalletConnect** provides a link between a **Wallet Address** and a user's **Game Account**. 
+There are two ways to get a **Session Key** from your wallet. You can connect with `QRCode` or using `WalletConnect.Instance.OpenMobileWallet`. The easiest way to connect is with `QRCode`.
```
Comment (Krayt78) [inline docs/Game/Unity/getting-started/01-connect.md:8]: Its not really the easiest way. It depends on the platform they want to build to.
QR Code should be used when building desktop-based applications
the Other way for android/ios 
we are working on a way for WebGL builds and standalone VR

**Review by Krayt78 — CHANGES_REQUESTED** [review]

File: docs/Game/Unity/getting-started/01-connect.md (hunk 1 above)
Comment (ddocs) [reply docs/Game/Unity/getting-started/01-connect.md:8]: Ok. Amending this.

**Review by Krayt78 — APPROVED** [review]

File: docs/Game/Unity/getting-started/01-connect.md [hunk 2]
```diff
@@ -3,25 +3,59 @@ title: 01 - Connect Wallet and Authenticate
...
+## 01 Connect with QRCode
+
+1. To connect with **QRCode**, use a ```QRCodeImage component```.
+This component already comes with a QRCode generator and a function that handles everything. 
+
+2. Call `UpdateQRCode(string url)` by giving it a URL. 
+You can then use `.SetImageActive(bool)` to activate/deactivate the QRCode.
+
+Here is the way it is implemented in our **Examples**.
+We get the `ConnectURL` from `walletconnect` Instance. Then we generate the `QRCode` from it and activate the QRCode's image so it can be scanned:
```
Comment (raproid) [inline docs/Game/Unity/getting-started/01-connect.md:24]: from a `walletconnect` instance?

File: docs/Game/Unity/getting-started/01-connect.md [hunk 3]
```diff
@@ -3,25 +3,59 @@ title: 01 - Connect Wallet and Authenticate
...
-3. Create an instance of a AnkrSDKWrapper class via AnkrSDKWrapper.GetSDKInstance(...) method after successful connection to your wallet.
+At this point, you are connected and any transactions that require the user signing a message will pop up in their MetaMask app on their phone. 
+
+:::tip
+
+Occasionally, the MetaMask mobile app does not pop up by itself. Should this be case, open the app manually.
+
+:::
+
+## 02 Accept Connection
```
Comment (raproid) [inline docs/Game/Unity/getting-started/01-connect.md:44]: I thought you said we're using sentence case. Why capitalize both words?

File: docs/Game/Unity/getting-started/01-connect.md (hunk 2 above)
Comment (ddocs) [reply docs/Game/Unity/getting-started/01-connect.md:24]: Done

**Review by raproid — APPROVED** [review]

File: docs/Game/Unity/getting-started/01-connect.md (hunk 3 above)
Comment (ddocs) [reply docs/Game/Unity/getting-started/01-connect.md]: Yep sentence case. Done.

**Pull request merged**